app/assets/images/payment_icons/monero.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 21.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="38" height="24" viewBox="0 0 80 80" style="enable-background:new 0 0 80 80;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#FFFFFF;}
+	.st1{fill:#FF6600;}
+	.st2{fill:#4C4C4C;}
+</style>
+<rect x="0" class="st0" width="80" height="80"/>
```

Comment:
Since the entire SVG is wider than it is tall this means there's transparant background on the sides, not a complete white background per the checklist.